wikimedia/wikipedia-ios
# Top read network fallback & in-memory images for snapshot

### Test Steps
1. Verify that you see top read content on a clean install without opening the app.
2. Smoke test image loading in top read widgets.
